perf.groovy

```diff
@@ -691,6 +691,109 @@ parallel(
     }
 }
 
+// Setup size-on-disk test
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        ['x64', 'x86'].each { arch ->
+            def architecture = arch
+            def newJob = job(Utilities.getFullJobName(project, "sizeondisk_${arch}", isPR)) {
+
+                // Set the label.
+                label('windows_server_2016_dotnet_sizeondisk')
```

Comment:
Is there a *windows_server_2016_dotnet_sizeondisk* machine pool (VMs) on Jenkins? If not, then you could use the same label than LinkBench/IlLink.
(`label(string)` tells Jenkins what is the machine pool where the leg will run on.)

Reply:
Changed it to 'windows_server_2016_clr_perf' if that is alright